src/types/circle.ts

```diff
@@ -81,16 +77,16 @@ export default /* GraphQL */ `
     isMember: Boolean!
 
     "Setting of this Circle."
-    setting: CircleSetting!
+    setting: CircleSetting! @auth(mode: "${AUTH_MODE.oauth}")
 
     "Invitations belonged to this Circle."
-    invites: Invites!
+    invites: Invites! @auth(mode: "${AUTH_MODE.oauth}")
 
     "Invitation used by current viewer."
     invitedBy: Invitation
 
     "Analytics dashboard."
-    analytics: CircleAnalytics!
+    analytics: CircleAnalytics! @auth(mode: "${AUTH_MODE.oauth}")
```

Comment:
I thought `@auth` can only be applied under `viewer`  🤦🏼

Reply:
You are right, will fix it.

Reply:
fixed